src/robot/running/model.py

```diff
@@ -57,7 +57,7 @@
 
 
 IT = TypeVar('IT', bound='IfBranch|TryBranch')
-BodyItemParent = Union['TestSuite', 'TestCase', 'UserKeyword', 'For', 'If', 'IfBranch',
+BodyItemParent = Union['TestSuite', 'TestCase', 'Keyword', 'For', 'If', 'IfBranch',
```

Comment:
This change isn't correct. `Keyword` can be a parent only in the result model and here in the running model `UserKeyword` is a valid parent.